packages/@stylexjs/eslint-plugin/__tests__/stylex-valid-styles-test.js

```diff
@@ -1857,5 +1857,89 @@ revert`,
         },
       ],
     },
+    {
+      code: `
+        import * as stylex from '@stylexjs/stylex';
+        const styles = stylex.create({
+          invalidStyle: {
+            margin: "10",
+            height: "10",
+          },
+        });
+      `,
+      errors: [
+        {
+          message:
+            'margin value must be one of:\n' +
+            'a number literal or math expression\n' +
+            'a non-numeric string\n' +
```

Comment:
These messages could align more with the formal CSS syntax for a given property, e.g., "a CSS <length>, <length-percentage>" etc. https://developer.mozilla.org/en-US/docs/Web/CSS/width#formal_syntax